### ISSUE TITLE
[Smoke Test] Url in format 1.1.1.1:1 open blank page

### ISSUE THREAD

**Issue by dakaizou** [mozilla-tw/Zerda#858]
Crash by following #809 STR (change http://... to 1.1.1.1:1)

Build: 1124
DuT: Samsung, advan

**Comment by dakaizou** [comment]
related to #290 #194

**Comment by chianger** [comment]
P0 for it's likely a regression

**Comment by walkingice** [comment]
No, it is different bug. Even I check out to an old version(ie. Sep 6th), we can still reproduce this "open 1.1.1.1:1 got blank page".

I tried to open from 1.1.1.1:1 ~ 1.1.1.1:25 on firefox desktop, I got `This address is restricted. This address uses a network port which is normally used for purposes other than Web browsing. Firefox has canceled the request for your protection.`

If above port 25, Firefox starts to reach host 1.1.1.1

Another funny(?) thing is that, although you see a blank page, you can still click it and fire click event to Homescreen.

**Comment by mTwTm** [comment]
Move this to decision pending based on walkingice's comment

**Comment by chianger** [comment]
should display the error page. but this is more of a corner case so it should not block.
